pkg/review/commit_review_status.go

```diff
@@ -152,9 +153,9 @@ type BigQueryBreakGlassIssueFetcher struct {
 	config *CommitApprovalPipelineConfig
 }
 
-func (bqif *BigQueryBreakGlassIssueFetcher) getBreakGlassIssues(ctx context.Context, author, timestamp string) ([]breakGlassIssue, error) {
-	issueQuery := GetBreakGlassIssueQuery(bqif.config.IssuesTable, author, timestamp)
-	items, err := bq.Query[breakGlassIssue](ctx, bqif.client, issueQuery)
+func (bqif *BigQueryBreakGlassIssueFetcher) getBreakGlassIssues(ctx context.Context, author, timestamp string) ([]*breakGlassIssue, error) {
+	issueQuery := BreakGlassIssueQuery(bqif.config.IssuesTable, author, timestamp)
+	items, err := bq.Query[*breakGlassIssue](ctx, bqif.client, issueQuery)
```

Comment:
I believe changing this to return pointers will cause this line to fail as the pointer will be initialized to nil:
https://github.com/abcxyz/github-metrics-aggregator/blob/main/pkg/review/bq/bq.go#L45

Reply:
https://github.com/abcxyz/github-metrics-aggregator/pull/182/commits/d04e6762b71c1820ee7c719e8eeb24b1d9b5f06d